tox.ini

```diff
@@ -1,11 +1,22 @@
 [tox]
 # We do not currently test with pypy because psycopg2 does not work there.
-envlist = py27,py37
+envlist =
+  py27
+  py37
+  lint
 
 [testenv]
-# Run a single test with e.g. tox -e py37 test/sqlalchemy/test_run_transaction.py
+deps =
+  -r test-requirements.txt
 commands =
-     py.test -p sqlalchemy.testing.plugin.pytestplugin {posargs:test/sqlalchemy}
-deps = --requirement=test-requirements.txt
-# For some reason pip fails to load the requirements file without this.
-setenv = LANG=en_US.utf-8
+  py.test -p sqlalchemy.testing.plugin.pytestplugin {posargs:test/sqlalchemy} -v
+
+[testenv:lint]
+skip_install = True
+deps =
+  flake8==3.6.0
+commands =
+  flake8 cockroachdb test
+
+[flake8]
```

Comment:
Huh, flake8 can read from tox.ini too? I'd rather put this in setup.cfg; I think that's more conventional.